plugins/modules/packaging/os/zypper_repository.py

```diff
@@ -186,6 +186,10 @@ def _repo_changes(realrepo, repocmp):
             valold = str(repocmp[k] or "")
             valnew = v or ""
             if k == "url":
+                cmd = ['rpm', '-q', '--qf', '%{version}', '-f', '/etc/os-release']
+                rc, stdout, stderr = module.run_command(cmd, check_rc=True)
```

Comment:
I think the command is pretty safe, but should I add a check that if `rc != 0` skip the whole replace?

Reply:
`check_rc=True` will already make the module fail, so `rc` will always be `== 0` here.

Reply:
Yes, I meant skip instead of fail, so no one notices this change :)

Reply:
Hmm, I think it's generally not a good idea to ignore errors, so it's probably better to not fail. But I have no idea how safe that command is :)

Reply:
Fine, I'll let it be as it is.
It's pretty safe IMO, I can also change it to use `grep` to retrieve version directly from `/etc/os-release`, but I still think this is better.